orbit/pkg/osquery/osquery.go

```diff
@@ -169,7 +169,15 @@ func (r *Runner) Execute() error {
 		if err := r.proc.Start(); err != nil {
 			return fmt.Errorf("start osqueryd: %w", err)
 		}
-		if err := r.proc.WaitOrKill(ctx, 10*time.Second); err != nil {
+
+		var killDelay time.Duration
+		if runtime.GOOS == "windows" {
+			killDelay = 100 * time.Millisecond
```

Comment:
Sorry, please remind me, why don't we give osqueryd 10s to graceful terminate before killing on Windows?

Looks like on Windows it sends the kill signal (`p.OsProcess().Signal(os.Kill)`), and then after a duration of `killDelay` it will try to kill the process with `p.OsProcess().Kill()`. Guessing there's a difference in those two ways of killing the process in Windows?

Reply:
I documented the rational behind this change [here](https://github.com/fleetdm/fleet/issues/8057#issuecomment-1277491360). I removed this logic in favour of addressing this and other required changes through a different PR.